listings/ch10-testing-cairo-programs/no_listing_09_integration_test/Scarb.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "adder"
-version = "0.1.0"
-edition = "2023_11"
+version = "0.2.0"
+edition = "2024_06"
```

```suggestion
version = "0.1.0"
edition = "2023_11"
```



it's not the current date :laughing: